script.js

```diff
@@ -1,4 +1,259 @@
+// There will be only two players. One human and one computer (for the Base solution).
+// The computer will always be the dealer.
+// Each player gets dealt two cards to start. (both face up, dealer has 1 card open and 1 card closed)
+// The player goes first, and decides if they want to hit (draw a card) or stand (end their turn).
+// The dealer has to hit if their hand is below 17.
+// Each players' score is the total of their card ranks. Jacks/Queen/Kings are 10. Aces can be 1 or 11.
+// The player who is closer to, but not above 21 wins the hand.
+
+//global variables
+var deck = [];
+var gameMode = "first round";
+var winningMode = ""; //blackjack, bigger number max 21
+var playerHand = []; //to reset
+var computerHand = [];
+var gameRound = 1;
+
+function getRandomNumber(max) {
+  return Math.floor(Math.random() * max);
+}
+function shuffleDeck() {
```

Comment:
Ideally, you want this function to have a parameter. Without a parameter, this function is very rigid and cannot shuffle any other decks with a different variable name